configs/en-sl-spring-2024.yml

```diff
@@ -0,0 +1,163 @@
+# The initial configuration was generated using:
+# task config-generator -- en sl --name spring-2024
+#
+# The documentation for this config can be found here:
+# https://github.com/mozilla/firefox-translations-training/blob/eeb21584f863bf2dd99259f15a9b5b25350b2271/taskcluster/configs/config.prod.yml
+experiment:
+  name: spring-2024
+  src: en
+  trg: sl
+  best-model: chrf
+  use-opuscleaner: 'true'
+  opuscleaner-mode: defaults
+  bicleaner:
+    default-threshold: 0.5
+    dataset-thresholds: {}
+  mono-max-sentences-src: 500_000_000
```

Comment:
We need to figure out a policy for monolingual data. We have full control of the mix in OpusTrainer now. but if it's say just 30M of original data I don't know if it makes sense to use 200M mono. Let's discuss live.

Reply:
We decided to use the current policy as news-crawl for English is ~200M sentences and it's fine to have some overhead for back-translations

Reply:
@gregtatum I remember now why I wanted to limit back-translations more for smaller original corpus. With the two-stage approach, we train for two epochs of back-translated first and it will take too long with 200M sentences and reach the second stage quite late. Maybe it's fine for prod training, but it wasn't convenient for the experiments